File: DiaryNutrition/app/src/main/java/ediger/diarynutrition/data/Meal.java
package ediger.diarynutrition.data;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "meal")
public class Meal {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "_id")
    private int id;

    private String name;

    public Meal(int id, String name) {
        this.id = id;
        this.name = name;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/ediger/diarynutrition/data/source/entities/Meal.kt
package ediger.diarynutrition.data.source.entities

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.Index
import androidx.room.PrimaryKey

@Entity(tableName = "meal")
data class Meal(var name: String,
           @field:ColumnInfo(defaultValue = "0") var user: Int = 0) {
    @PrimaryKey(autoGenerate = true)
    var id = 0

    override fun toString(): String = name
}